opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -2142,8 +2224,6 @@ const prepareRelation = (thing) => {
   });
   return R.pipe(
     R.assoc('connections', connections),
-    R.dissoc(INTERNAL_TO_FIELD),
-    R.dissoc(INTERNAL_FROM_FIELD),
```

Comment:
I think it's because `prepareRelation` is called only in `prepareElementIndexing` and inside we already call `prepareEntity` first which does this same operation